SDWebImage.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please don't submit this auto generated files into the repo. You can ignore this from git-config. Or maybe you can modify the `.gitignore` files to add this line of file into it.

Reply:
Ok, thanks @dreampiggy. If you'd like I can open a separate PR to add this file or ignore it. My understanding is files in xcshareddata are intended to be version controlled.

Reply:
Typically you're right. But this `IDEWorkspaceChecks.plist` from Xcode 9.3 is a exception. This files only contains the 32-bit warning for Xcode now. So it's useless for most projects.

If we want to use it, I can submit this files again. Sorry for bother.